src/ui/buttons/buttonDesigns/alternateSlideLogoDesign.jsx

```diff
@@ -0,0 +1,209 @@
+        ANIMATION_LABEL_CONTAINER:  'personalized-label-container',
+        PAYPAL_BUTTON_LABEL:        CLASS.BUTTON_LABEL
+    }
+};
+
+export function AlternateSlideLogoComponent({ designLabelText } : ContentOptions) : ChildType {
+    const CONTAINER_CLASS = ALTERNATE_SLIDE_LOGO_CONFIG.cssClasses.ANIMATION_CONTAINER;
+    const LABEL_CLASS = ALTERNATE_SLIDE_LOGO_CONFIG.cssClasses.ANIMATION_LABEL_CONTAINER;
+   
+    return (
+        <Fragment>
+            <div class={ LABEL_CLASS } data-design-experiment='104530'> <span>{designLabelText}</span></div>
+            <style innerHTML={ `
+                .${ CLASS.DOM_READY } .${ CONTAINER_CLASS } img.${ LOGO_CLASS.LOGO }{
+                    position: relative;
+                }
+                
+                .${ CONTAINER_CLASS } .${ LABEL_CLASS } {
+                    position: fixed;
+                    opacity: 0; 
+                    
+                }
+            ` } />;
+        </Fragment>
+    );
+}
+
+// Returns label container if the button sizes match
+export const getAlternateSlideLogoProps = function(document : Object, configuration : Object) : AnimationProps | null {
+    let paypalLogoStartingLeftPosition = null;
+    const { ANIMATION_CONTAINER, ANIMATION_LABEL_CONTAINER, PAYPAL_BUTTON_LABEL, PAYPAL_LOGO } = configuration.cssClasses;
+    const { min, max, smalMax, runOnce } = configuration;
+    // get the animation main container to force specificity( in css ) and make sure we are running the right animation
+    const designContainer = (document && document.querySelector(`.${ ANIMATION_CONTAINER }`)) || null;
+    if (!designContainer) {
+        return null;
+    }
+
+    // return null if animation should not be played for the button size
+    const animationContainerWidth = designContainer.offsetWidth;
+    if (animationContainerWidth < min || animationContainerWidth > max) {
+        // remove label element from dom
+        designContainer.querySelector(`.${ ANIMATION_LABEL_CONTAINER }`).remove();
+        return null;
+    }
+
+    // get the label container that animation will be applied to
+    const paypalLabelContainerElement = designContainer.querySelector(`.${ PAYPAL_BUTTON_LABEL }`) || null;
+    // get starting position for element so it doesn't jump when animation begins
+    const paypalLogoElement = (paypalLabelContainerElement && paypalLabelContainerElement.querySelector(`.${ PAYPAL_LOGO }`)) || null;
+    
+    if (paypalLogoElement && paypalLabelContainerElement) {
+        paypalLogoStartingLeftPosition = `${ (paypalLogoElement.offsetLeft / paypalLabelContainerElement.offsetWidth) * 100 }`;
+    } else {
+        return null;
+    }
+    // increase font-size for medium button sizes
+    const labelTextFontSize = animationContainerWidth >= smalMax ? 11  : 8;
+
+    if (runOnce) {
+        // enable reverse logo animation on mouser out
+        designContainer.addEventListener('mouseleave', () => {
```

Comment:
Instead of 2 separate animations, could you have it as one animation that runs once onHover? I understand this is a design change so it's up to Mercedes